File: Practicas/Practica11/ejercicio2.py
#Mayor de una lista de numeros


print("Encontrando el numero mayor de una lista de numeros\n")

lista =[3,2,8,7,6,3,4,0,1]
mayor = 0
for i in lista:
    if mayor <= i:
        mayor = i
print(f"La lista es: {lista}")
print(f"El numero mayor es: {mayor}")

File: Practicas/Practica11/ejercicio2.c
//Mayor de una lista de numeros

#include<stdio.h>

int main(){
	int lista[50]={0}, i=0, cont=0;
	int mayor = lista[0];
	
	printf("Encontrando el numero mayor de una lista de numeros\n");
	printf("¿Cuantos numeros deseas ingresar?");
	scanf("%d", &cont);
	
	for(i; i<cont; i++){
		printf("\nIngresa el numero: ");
		scanf("%d", &lista[i]);
		if(mayor <= lista[i]){
			mayor = lista[i];
		}
	} 
	printf("El mayor de la lista es: %d", mayor);
}
